grpc/pb-marshalling.go

```diff
@@ -360,10 +361,57 @@ func pbToAuthz(pb *rapb.Authorization) (core.Authorization, error) {
 	}, nil
 }
 
-func registrationValid(reg *rapb.Registration) bool {
+func registrationValid(reg *corepb.Registration) bool {
 	return !(reg.Id == nil || reg.Key == nil || reg.Agreement == nil || reg.InitialIP == nil || reg.CreatedAt == nil || reg.Status == nil || reg.ContactsPresent == nil)
 }
 
-func authorizationValid(authz *rapb.Authorization) bool {
+func authorizationValid(authz *corepb.Authorization) bool {
 	return !(authz.Id == nil || authz.Identifier == nil || authz.RegistrationID == nil || authz.Status == nil || authz.Expires == nil)
 }
+
+func certificateValid(cert *corepb.Certificate) bool {
+	return !(cert.RegistrationID == nil || cert.Serial == nil || cert.Digest == nil || cert.Der == nil || cert.Issued == nil || cert.Expires == nil)
+}
+
+func sctToPB(sct core.SignedCertificateTimestamp) *sapb.SignedCertificateTimestamp {
```

Comment:
Can you add unit tests for `sctToPB`, `pbToSCT`, and `certToPb` to `pb-marshalling-test.go`?